clarity_ext/dilution_strategies.py

```diff
@@ -0,0 +1,137 @@
+# These values, residing in SingleTransfer objects, are calculated in
```

Comment:
Looks good! I think dragging different strategies out of the service like this, following the single responsibility principle, and then injecting them into the service (as opposed to e.g. using class hierarchies) is definitely the way to go.